charts/admission-controller/templates/scanner/podmonitor.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.podMonitors.scanner.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  annotations: {{ .Values.podMonitors.scanner.annotations | toYaml | nindent 4 }}
+  labels: {{ .Values.podMonitors.scanner.labels | toYaml | nindent 4 }}
+  name: admission-controller-scanner
+spec:
+  selector:
+    matchLabels:
+      {{- include "admission-controller.scanner.selectorLabels" . | nindent 6 }}
+  podMetricsEndpoints:
+    - targetPort: {{ .Values.scanner.service.port }}
+      path: {{ .Values.podMonitors.scanner.endpoint }}
```

Let's remove this option, because the `/metrics` endpoint is currently not configurable in code.
```suggestion
      path: "/metrics"
```